Resources/Prototypes/Maps/helios.yml

```diff
@@ -0,0 +1,72 @@
+- type: gameMap
+  id: Helios
+  mapName: 'Helios'
+  mapPath: /Maps/helios.yml
+  minPlayers: 75
+  stations:
+    Helios:
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Helios {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'TG'
```

Comment:
Is this a TG station?

Reply:
I forgot to change the prefix to 14, let me go fix that now.

Reply:
You know, I've never bothered changing the prefix. We should probably make it DV on all our maps...?